src/ontology/edits/oeo-model.omn

```diff
@@ -3392,6 +3392,28 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/878",
         OEO_00000078
     
     
+Class: OEO_00240022
+
+    Annotations: 
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1052
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1078
+
+rework module structure 
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1592
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652
+
+made class a 'region of relevance'
```

Comment:
For the `protected area`:
Rather than _made class a 'region of relevance'_ I would prefer the info that this has moved here from oeo-physical. What do you think, @stap-m ?